frontend/src/components/regions/UsersAndGroups/Presentation.js

```diff
@@ -19,23 +22,41 @@ import LaunchIcon from "@material-ui/icons/Launch";
 
 import { Link } from "react-router-dom";
 import { routes } from "../../../routes";
-const useStyles = makeStyles({
+
+const useStyles = makeStyles((theme)=> ({
   marginsAlignment: {
     marginTop: "8px",
     marginLeft: "16px",
   },
   textBoxSize: {
     width: "62%",
   },
-});
+  warning: {
+    color: theme.palette.warning.dark
+  },
+  error: {
+    color: theme.palette.error.dark
+  },
+  default: {
+    color: theme.palette.primary
+  }
+
+}));
+
 const TeamSummaryStats = ({ summaryStats }) => {
-  const oldestOpenPrTime = summaryStats.oldestOpenPrTime
-    ? new Date(summaryStats.oldestOpenPrTime)
-    : null;
+  const openPrAge = summaryStats.oldestOpenPrTime.slice(0, 10); 
+  const tildeReviewAge = summaryStats.oldestCardInReviewTime.slice(0, 10);
+  const classes = useStyles();
+
+  const prStatusClassName = 
+  getPrStatus(summaryStats.oldestOpenPrTime) === "error" ? classes.error : 
+  getPrStatus(summaryStats.oldestOpenPrTime) === "warning" ? classes.warning : 
```

Comment:
This is better:

`const prStatusClassName = classes[getPrStatus(summaryStats.oldestOpenPrTime)]`

One of the great things about objects is that we can access values by key.